doc-content/enterprise-only/cases/case-management-defining-a-stage-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='case-management-defining-a-stage-proc-{context}']
 = Defining a stage
 
-You can use the process designer to model a stage in BPMN2. Stages are a way of grouping related tasks in a way that clearly defines activities that, if the stage is activated, must complete before the next stage of the case commences. For example, the *IT_Orders* case definition can also be defined using stages in the following way:
+You can use the process designer to model a stage in BPMN2. You can use stages to group related tasks of clearly defines activities that, if the stage is activated, must complete before the next stage of the case commences. For example, the *IT_Orders* case definition can also be defined using stages in the following way:
```

Comment:
"to group related tasks of clearly defines activities" does not look right. Now, "defines" might be a typo - but "tasks of clearly defined activities" does not fully make sense to me. Maybe "group related tasks INTO clearly defined activities"?